backend/app/models/location_type.rb

```diff
@@ -2,7 +2,7 @@ class LocationType
   include EnumModel
 
   TYPES = %w[
-    state
+    country
```

Comment:
I am renaming `state` to `country` so this keeps naming convention used at design